DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/Startup.cs

```diff
@@ -50,6 +52,10 @@ public void Configuration(IAppBuilder app)
 
             app.CreatePerOwinContext<ApplicationUserManager>(ApplicationUserManager.Create);
 
+            FluentValidationModelValidatorProvider
+                .Configure(config,
+                    provider => provider.ValidatorFactory = new FluentValidatorFactory(container));

```

Comment:
I don't know how to do this in simple way yet.
There is an example in FluentValidation wiki for MVC with lambda. https://github.com/JeremySkinner/FluentValidation/wiki/h.-MVC

FluentValidationModelValidatorProvider.Configure(provider => {
  provider.ValidatorFactory = new MyCustomValidatorFactory();
});

But I will try to find another way.